packages-next/types/src/config/index.ts

```diff
@@ -36,6 +36,10 @@ export type KeystoneConfig = {
     enableNextJsGraphqlApiEndpoint?: boolean;
     /** Enable Prisma+SQLite support */
     prismaSqlite?: boolean;
+    /** Creates a file at `node_modules/.keystone/api` with a `lists` export */
+    generateNodeAPI?: boolean;
```

Comment:
Could we make sure that the `config.mdx` has at least one sentence about each of these new options 👍